predictor-server/src/main/java/ru/dbzukunft/predictor/server/util/dto/AlphaVantageApiDTO.java

```diff
@@ -0,0 +1,19 @@
+package ru.dbzukunft.predictor.server.util.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+import java.io.Serializable;
+import java.util.Map;
+
+@Builder
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+public class AlphaVantageApiDTO implements Serializable {
+    @JsonProperty(value = "Meta Data")
+    Map<String, String> metaData;
+    @JsonProperty(value = "Time Series (5min)")
```

Comment:
Seems this gonna be broken if we change **alphavantage.interval** param from **5min** to other value